src/l10n/ru.js

```diff
@@ -0,0 +1,354 @@
+            'Revision restored'                   : 'Ревизия восстановлена',
+            'Restoring revision failed'           : 'Не удалось восстановить выбранную ревизию',
+            'Do you want to restore the revision?': 'Хотите восстановить данную ревизию?',
+            'Shared with me'                      : 'Поделились со мной',
+            'Shared by me'                        : 'Мною предоставлен доступ',
+            'Statistics'                          : 'Статистика',
+            'Created on'                          : 'Создано',
+            'Last updated'                        : 'Обновлено',
+            '{count} revisions'                   : 'Ревизий: {count}',
+            'Shares'                              : 'Общий доступ',
+            '{count} shares'                      : 'Предоставлено в общий доступ: {count}',
+            'More'                                : 'Развернуть | Свернуть',
+            'Backup and Restore'                  : 'Архивирование и восстановление',
+            'Browser Extension'                   : 'Расширения для браузеров',
+            'Handbook'                            : 'Руководство пользователя',
+            'Search user'                         : 'Поиск пользователей',
+            'Set expiration date'                 : 'Установить финальную дату',
+            'Expires {date}'                      : 'Срок истекает {date}',
+            'Choose expiration date'              : 'Выбрать финальную дату',
+            'Please choose a date in the future'  : 'Выберите финальную дату',
+            'Invalid date'                        : 'Неверная дата',
+            'The user {uid} does not exist'       : 'Пользователь {uid} не существует',
+            'Unable to share password: {message}' : 'Невозможно предоставить доступ: {message}',
+            'Toggle write permissions'            : 'Переключить разрешение на запись',
+            'Toggle share permissions'            : 'Переключить возможность поделиться',
+            'Stop sharing'                        : 'Отключить доступ',
+            'Date'                                : 'Дата',
+            'Default Cache (0 files, 0 B)'        : 'Кэш по умолчанию (0 файлов, 0 B)',
+            'Avatars Cache (0 files, 0 B)'        : 'Кэш аватаров (0 файлов, 0 B)',
+            'Favicon Cache (0 files, 0 B)'        : 'Кэш фавиконов (0 файлов, 0 B)',
+            'Preview Cache (0 files, 0 B)'        : 'Кэш превьюшек (0 файлов, 0 B)',
+            'Passwords Cache (0 files, 0 B)'      : 'Кэш паролей (0 файлов, 0 B)',
+            'Backup or export'                    : 'Создание резервной копии или экспорт',
+            'Choose Format'                       : 'Выбор формата',
+            'Please choose'                       : 'Выберите формат',
+            'Database Backup'                     : 'Резервная копия в БД',
+            'Predefined CSV'                      : ' CSV',
```

Comment:
I think something is missing here

Reply:
I leaved this empty because did not find a good russian equivalent for "Predefned".
I think anyone who understand what CSV is will not get stuck when see just "CSV".
If 'Predefined CSV' is close by meaning to 'Custom CSV' you can use Russian 'Пользовательский CSV".

Reply:
Predefined CSV is the opposite of Custom CSVs.
It's a CSV file which contains all fields and can be used to import it again without having to map the fields manually. So it could also be named "Standarized CSV" or "Importable CSV". Otherwise i would suggest to just use "Passwords CSV".